samples/demos/end_to_end/README.md

```diff
@@ -0,0 +1,73 @@
+End to End Sample Application
+=============================
+
+This example demonstrates how to use the XBee Gateway to transmit data from
+XBee nodes of the network to Digi Remote Manager and vice versa.
+
+The example reads every 10 seconds the temperature and humidity from the board's
+I2C sensor, sends the readings to the gateway and goes to sleep until the
+next cycle. It also processes incoming data messages from the gateway to
+change the sampling rate or disable the temperature/humidity service.
+
+Requirements
+------------
+
+To run this example you need:
+
+* One XBee 3 radio module with MicroPython support.
+* One carrier board for the radio module (XBIB-C board).
+* An XBee Gateway device.
+* A Digi Remote Manager account with your XBee Gateway added to it.
+  Go to https://myaccount.digi.com/ to create it if you do not have one.
+* Another instance of PyCharm with the 'End to End' sample for Python loaded
+  (located in the *DEMOS* category).
+
+Setup
+-----
+
+Make sure the hardware is set up correctly:
+
+1. Plug the XBee 3 radio module into the XBee adapter and connect it to your
+   computer's USB port.
+
+Run
+---
+
+The example is already configured, so all you need to do is build and launch
+the project. Then, launch the **End to End** sample for Python in the XBee
+Gateway and follow the instructions explained in that sample's README file.
```

Comment:
Where does someone find that code?  It isn't in this PR.  If it's in another repo, can you include a link to that repo?

Reply:
We are generalizing the PyCharm plugin to work not only with XBees (MicroPython) but also with gateways, ConnectCores, routers... (Python). Each platform family will have its own repo like this one in GitHub, but since we are still working on it, for the moment the XBee gateway repo is in Stash.

My idea was to wait to merge this sample until we release the plugin and all the gateway stuff, so I can add a link to the other repo here when it is public.